CHANGES.md

```diff
@@ -1,3 +1,66 @@
+## edge-19.4.5
+
+**Significant Update**
+
+As of this edge release the proxy auto-injector component is always installed.
+To have the auto-injector inject a pod you still have the option to manually
+add the `linkerd.io/inject: enable` annotation into the pod spec, or at the
+namespace level to have all your pods be injected by default.
+With this release the behaviour of the `linkerd inject` command changes, as
```

Comment:
We need a stylesheet, but in the past we have always used the non-British "behavior" spelling

Reply:
Bloody hell, I keep on making those kind of mistakes! :gb:

Reply:
It's my least favourite kind of mistake because it's just a nit!